shaders/src/main/glsl/samples/310es/mandelbrot_blurry_intfunctions.frag

```diff
@@ -0,0 +1,70 @@
+#version 310 es
+
+/*
+ * Copyright 2018 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+precision highp float;
+
+layout(location = 0) out vec4 _GLF_color;
+
+uniform vec2 resolution;
+
+vec3 pickColor(int i) {
+  return vec3(float(i) / 50.0, float(i) / 120.0, float(i) / 140.0);
+}
+
+vec3 mand(float xCoord, float yCoord) {
+  float height = resolution.y;
+  float width = resolution.x;
+
+  float c_re = 0.8*(xCoord - width/2.0)*4.0/width - 0.4;
+  float c_im = 0.8*(yCoord - height/2.0)*4.0/width;
+  float x = 0.0, y = 0.0;
```

Comment:
The way built-ins are used below (in this `mand` function) are probably a bit too simple. I would try to complicate one of them to not just take literals.

Reply:
I've complicated it with some uniform dependent expressions.